lib/matplotlib/backend_bases.py

```diff
@@ -1437,21 +1422,17 @@ class PickEvent(Event):
     A pick event, fired when the user picks a location on the canvas
     sufficiently close to an artist.
 
-    Attrs: all the :class:`Event` attributes plus
+    Attrs: all the `Event` attributes plus
 
     Attributes
     ----------
     mouseevent : `MouseEvent`
-        the mouse event that generated the pick
-
+        The mouse event that generated the pick.
     artist : `matplotlib.artist.Artist`
-        the picked artist
-
+        The picked artist.
     other
-        extra class dependent attrs -- e.g., a
-        :class:`~matplotlib.lines.Line2D` pick may define different
-        extra attributes than a
-        :class:`~matplotlib.collections.PatchCollection` pick event
+        Extra class dependent attrs -- e.g., a `~.Line2D` pick may define
```

Comment:
~~~
Additional attributes may be present depending on the type of the
picked object; e.g. a `~.Line2D` pick may provide other additional
information than a `~PatchCollection` pick.
~~~